net: socket: tcp: net.socket.tcp overflow during building

**Describe the bug**
Test `tests/net/socket/tcp/net.socket.tcp` reports overflow during building.

Observed for ` nrf5340dk/nrf5340/cpuapp/ns`.

**To Reproduce**
Steps to reproduce the behavior:
1. have ` nrf5340dk/nrf5340/cpuapp/ns` connected
2. go to your zephyr dir
3. call `./scripts/twister -T tests/net/socket/tcp/ -p nrf5340dk/nrf5340/cpuapp/ns --device-testing --device-serial /dev/ttyACM1 -v --inline-logs --west-flash="--erase"`
4. See console output with error

**Expected behavior**
Valid console output

**Impact**
Not clear

**Logs and console output**
```
opt/zephyr-sdk/arm-zephyr-eabi/bin/lib/gcc/arm-zephyr-eabi/12.2.0/arm-zephyr-eabi/bin/ld.bfd: zephyr/zephyr_pre1.elf section `kobject_data' will not fit in region `FLASH'
opt/zephyr-sdk/arm-zephyr-eabi/bin/lib/gcc/arm-zephyr-eabi/12.2.0/arm-zephyr-eabi/bin/ld.bfd: region `FLASH' overflowed by 128 bytes
collect2: error: ld returned 1 exit status
ninja: build stopped: subcommand failed.
```

**Environment (please complete the following information):**
 - OS: Ubuntu 20.04.1 LTS
- Toolchain Zephyr SDK 0.16.8
- Commit SHA or Version used: `v4.0.0-289-ga6cca606dbf4`

--- a/tests/net/socket/tcp/testcase.yaml
+++ b/tests/net/socket/tcp/testcase.yaml
@@ -1,6 +1,7 @@
 common:
   depends_on: netif
   min_ram: 32
+  min_flash: 194
   tags:
     - net
     - socket

